src/gui/input_mac.cpp

```diff
@@ -42,19 +43,45 @@ QString GetModifierPrefix(Qt::KeyboardModifiers mod) noexcept
 	return modprefix;
 }
 
+// Fixup Modifiers, Apples OPTION key
+Qt::KeyboardModifiers de_DE(Qt::KeyboardModifiers mod, QChar c) noexcept
+{
+ 	std::vector<QChar> removeAltCharList  { '[', ']', '|', '{', '}', '~', '@', '\'' };
+	if(mod & Qt::AltModifier) {
+		if (std::find(removeAltCharList.begin(), removeAltCharList.end(), c) != removeAltCharList.end()) {
+			mod &= ~Qt::AltModifier;
+		}
+	}
+	return mod;
+}
 QKeyEvent CreatePlatformNormalizedKeyEvent(
 	QEvent::Type type,
 	int key,
 	Qt::KeyboardModifiers mod,
-	const QString& text) noexcept
+	const QString& text,
+	QLocale* locale) noexcept
 {
+	
+	
+	
 	if (!text.isEmpty())
 	{
 		const QChar c{ text.at(0) };
 		if (c.unicode() >= 0x80 && c.isPrint()) {
 			mod &= ~Qt::AltModifier;
 		}
+		
+		// German/Austrian/Swiss MacOS QWERTZ keyboard, remove AltModifier (OPTION KEY)
+		if(locale != nullptr) {
+			if(locale->name() == "de_DE" || locale->name() == "de_AT" || locale->name() == "de_CH") {
```

Comment:
Totally optional: `locale->name().startsWith("de_", /*? Qt::CaseSensitive*/`

Something to consider if this applies to any german layout... Not strictly necessary.

Perhaps we only want call out certain german layouts?

Reply:
i think its good to do with de_ - and see if any regression pops up.